### ISSUE TITLE
[webcontent] Browser has multiple issues while using comment sections on several sites.

### ISSUE THREAD

**Issue by objecttothis** [mozilla-mobile/fenix#17025]
## Site with issue and any steps to reproduce
foxnews.com is just one example.  Click on any article link and go down to the comments section.  Initially the browser pops up a message telling you that you need to login to comment.  Click OK and it takes you to login screen which for the most part seems to behave, but once you login and return to any article and begin typing comments that is where the fun begins.

### Expected behavior
Be able to type comments without interruption to the typing or autocorrect.

### Actual behavior
During typing, the autocorrect search suggestions flicker and every few letters it completely misses or overwrites the word with.  It's almost as if something is trying to intervene.  I thought perhaps it was Google Board however this behavior on the same site is not exhibited in Chrome or Edge for Android.

### Does toggling Tracking Protection fix the issue? (Press the shield icon in the toolbar while on the site to see toggle)
No

### Can you reproduce in Chrome (or other non-Mozilla browser)?

No, it cannot be reproduced in other browsers, but it can be reproduced in Firefox for Android and Firefox Beta for Android apps.

### Device information

* Android device: OnePlus 6 Android 10 (OxygenOS 10.3.0)
* Fenix version: 201213 17::01 (Build #2015781195)

**Comment by ale82to** [comment]
I ve always had this issue in every fenix release shame has not been fixed yet

**Comment by objecttothis** [comment]
Glad I'm not taking crazy pills and at least someone else is experiencing it.

**Comment by Mugurell** [comment]
https://user-images.githubusercontent.com/11428869/106622246-135acd00-657c-11eb-962a-46384b35ebba.mp4

Tried to do a quick test but cannot reproduce.

@objecttothis If fhis is still an issue can you please post a video recording of what exactly happens?

**Comment by objecttothis** [comment]
I took the screenshot and I'm trying to post it but for some reason the wysiwyg is editing it out of my response.

**Comment by objecttothis** [comment]
Here you go. Turns out that the original file was too large.  I compressed it and it uploaded.


https://user-images.githubusercontent.com/17935339/106630576-62f9c280-6595-11eb-95fa-8fa1d276c0bb.mp4

**Comment by Mugurell** [comment]
Thank you @objecttothis!

This does seem like an issue that would be possible in the app but I cannot reproduce (tested with youtube comments).

Can I ask you to temporarily disable any addons & use a keyboard without autocomplete just to leave out any other parts that might influence this behavior and post the results after?

**Comment by objecttothis** [comment]
> Thank you @objecttothis!
> 
> This does seem like an issue that would be possible in the app but I cannot reproduce (tested with youtube comments).
> 
> Can I ask you to temporarily disable any addons & use a keyboard without autocomplete just to leave out any other parts that might influence this behavior and post the results after?

It does not do this with every site's comment sections. I dont experience it with youtube's comment section either. It always happens on Foxnews' comment section. They appear to use OpenWeb for their comments. I cant remember if it was happening with Disqus or not. my guess is that it's the browser's handling of some JavaScript that is causing the issue. This would explain why it's not every comment section on all sites any why other browsers dont have the issue even with the same keyboard.

That said I disabled then removed dark reader. I also went into the GBoard settings and disabled anything to do with spellcheck or autocorrect. The browser's handling of comment sections on foxnews articles was subjectively even worse (presumably because autocorrect wasn't undoing some of the browser's wonkiness). While typing the page was even scrolling up and down on me.

If this wasn't perfectly fine in other browsers on the same sites I would assume it was the site causing it.
